rebar.lock

```diff
@@ -107,11 +111,7 @@
   {git,"git://github.com/lasp-lang/time_compat.git",
        {ref,"6007f68892104ebb6fa2366cebf9d928d8856273"}},
   0},
- {<<"unicode_util_compat">>,{pkg,<<"unicode_util_compat">>,<<"0.2.0">>},2},
- {<<"vernemq_dev">>,
-  {git,"git://github.com/erlio/vernemq_dev.git",
-       {ref,"087fb0f84a2a848b71cc5e2391c4a3bac916a52f"}},
```

Comment:
looks good for now but need to keep in mind that pre 1.1.1 will need to use older versions of `vernemq_dev` i why i suggest to use git tags for it.

Reply:
yes, after the erlio/vernemq_dev#1 PR has been approved then we'd need to fixup things in this PR